helm/conjur-app-deploy/charts/app-secrets-provider-standalone/tests/test_app_secrets_provider_standalone_test.yaml

```diff
@@ -0,0 +1,67 @@
+suite: test test_app_secrets_provider_init
```

Comment:
`suite` here points to `test_app_secrets_provider_init` - assume we'd like `test_app_secrets_provider_standalone`

Reply:
Good catch, thanks! Will fix.